### PR TITLE
add guard to avoid .get() of absent value

### LINKED ISSUE #4: Bug: parsing error for skos ontology
I got a parsing error when parsing the skos ontology [ISOCOUNTRIES](https://biodiv-onto.gfbio.dev/ontologies/ISOCOUNTRIES)

Error message: owlapi.wrapper.OntologyParser - Optional.get() cannot be called on an absent value

Steps to reproduce:
1. clone repository master branch
2. compile: `mvn clean package`
3. java -DentityExpansionLimit=2500000 -Xmx10240M -jar ./target/owlapi-wrapper-1.4.2-SNAPSHOT.jar -m path/to/skos-ontology.rdf -o .. -r true

### PR DESCRIPTION
**Pull request by RaimiSol**
Fix #4 by adding a guard.

### CODE REVIEW

**Comment by jonquet** [comment]
Adding @syphax-bouazzouni to this... maybe we can ask @jvendetti too. 
Not sure about the issue.

**Comment by syphax-bouazzouni** [comment]
Hi @RaimiSol,

Thanks for this PR; some remarks 

1. You need to give more details about the issue and how you solved it 
Here, the issue was that resources (SKOS or OWL) with no <owl:ontology> clause defined does not parse and raise catched exception in the `addOntologyImports` function.

Your solution is to not call the `addOntologyImports` function if the ontology is anonymous.

2. About the code, you pushed a lot of linting changes (removing whitespace, ...), that are mixed with your real fix.

File: src/main/java/org/stanford/ncbo/owlapi/wrapper/OntologyParser.java [hunk 1]
```diff
@@ -204,29 +212,35 @@ private void addOntologyIRI(OWLOntology sourceOnt) {
...
-		IRI ontologyIRI = sub.get();
-		// Get imports and add them as omv:useImports
-		OWLAnnotationProperty useImportProp = fact.getOWLAnnotationProperty(IRI.create(USE_IMPORTS_IRI));
-		for (OWLOntology imported : sourceOnt.getImports()) {
-			if (!imported.getOntologyID().isAnonymous()) {
-				log.info("useImports: " + imported.getOntologyID().getOntologyIRI().get());
-				OWLAnnotationAssertionAxiom useImportAxiom = fact.getOWLAnnotationAssertionAxiom(useImportProp, ontologyIRI, imported.getOntologyID().getOntologyIRI().get());
-				this.targetOwlManager.addAxiom(targetOwlOntology, useImportAxiom);
+	private void addOntologyImports(OWLDataFactory fact, OWLOntology sourceOnt) {
+		if (!sourceOnt.getOntologyID().isAnonymous()) {
```
Comment (syphax-bouazzouni) [inline src/main/java/org/stanford/ncbo/owlapi/wrapper/OntologyParser.java]: a better syntax would be this 
```java
        Optional<IRI> sub = sourceOnt.getOntologyID().getOntologyIRI();
        IRI ontologyIRI;

        if(!sub.isPresent()){
            return;
        }
        ...
```

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
You need to remove the linting changes

**Comment by RaimiSol** [comment]
@syphax-bouazzouni thank you for the remarks. I updated the commit based on your suggestions.

**Review by syphax-bouazzouni — APPROVED** [review]

**Comment by syphax-bouazzouni** [comment]
Great thanks for the contribution.

**Pull request merged**